docs/deployment_patterns/how_to_use_great_expectations_in_flyte.md

```diff
@@ -0,0 +1,202 @@
+---
+title: How to Use Great Expectations in Flyte
+---
+import Prerequisites from '../guides/connecting_to_your_data/components/prerequisites.jsx'
+
+*This piece of documentation was authored by [Samhita Alla](https://www.linkedin.com/in/samhita-alla) and the [Flyte](https://flyte.org/) team*
+
+This guide will help you run a Great Expectations in [Flyte](https://flyte.org/)
+
+<Prerequisites>
```

Comment:
@talagluck I copied this over from the airflow example. Let me know if we need these and if we should remove or add any prerequisites